pkg/gadgets/execsnoop/tracer/core/bpf/execsnoop.bpf.c

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+#include <vmlinux/vmlinux.h>
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include "execsnoop.h"
```

Comment:
I think this file does not exist in this commit and is added in next commit.
I am OK with the actual situation (as nothing is provided to compile this file for the moment so no error can be generated) but if you can add this line in next commit it would be perfect!

Also, can you please add the SHA1 of the last bcc commit which modifies this file so we keep a trace of it?